docs/custom-webview-android.md

```diff
@@ -46,12 +51,43 @@ public class CustomWebViewManager extends ReactWebViewManager {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+@ReactModule(name = CustomWebViewManager.name)
+class CustomWebViewManager : ReactWebViewManager() {
+    protected class CustomWebViewClient : ReactWebViewClient()
+    protected class CustomWebView(reactContext: ThemedReactContext?) : ReactWebView(reactContext)
+
+    protected fun createReactWebViewInstance(reactContext: ThemedReactContext?): ReactWebView {
```

Comment:
The fact that here the is no `override` looks odd to me. Have you tested this code?